### ISSUE TITLE
Raw Configuration Parameters wont save for powernode-6 devices

### ISSUE THREAD

**Issue by jghaanstra** [athombv/com.greenwavesystems#1]
**What is your Homey version (Settings → System)?**
0.9.2

**What is your app version?**
Greenwave 1.0.0

**What did you try to achieve?**
Edit the raw configuration parameters for two of my Greenwave Powernode 6 devices. I tried setting the parameter 1,1,255 (keep alive time out to avoid green blinking lights)

**Please include detailed steps how we can reproduce the issue**
Edit the configuration of the Greenwave Powernode 6 main device. Set the configuration for the Raw Configuration Parameter to 1,1,255 and hit the save button.

**What was the result?**
The API request timing out with a popup with the message "True" but no changes to the raw configuration parameters.

**What did you expect as result?**
Saved raw configuration parameters.

**Any other remarks**
Nope

**Comment by JPe2** [comment]
My module only blinks green if communication is broken, at normal operation there is no blinking.
Only the ON sign is is steady green when one or more sockets are switched ON !!
On older Homey versions I also had blinking green. (my two cents)

**Comment by jghaanstra** [comment]
The blinking occurs if the polling frequency is higher than the keep alive time. The default polling frequency is 120 seconds and the default keep alive time is 2 minutes. So by default the green blinking does indeed not occur. But I have set my polling frequency higher to reduce the strain on the zwave chip.

But I have also created a custom version of the app where I can set the keep alive time as a device setting so there is no need to use the raw configuration parameters. It a workaround for this issue which works really good.. There is a pull request waiting to be reviewed to have this incorporated into the app available trough the app store.
